cob_bringup/robots/cob4-1.xml

```diff
@@ -61,6 +61,10 @@
 		<include file="$(find cob_bringup)/tools/emergency_stop_monitor.launch" >
 			<arg name="robot" value="$(arg robot)" />
 		</include>
+		<include file="$(find cob_bringup)/tools/battery_light_monitor.launch" >
+			<arg name="robot" value="$(arg robot)" />
+			<arg name="light_service" value="/light_torso/set_light"/>
```

Comment:
to be consistent with other monitors, we should move the config into a yaml file and not specify it in the launch file.